react-insta-clone/src/components/PostsPage/PostsPage.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import dummyData from '../../dummy-data';
+import SearchBar from '../SearchBar/SearchBar';
+import PostContainer from '../PostContainer/PostContainer';
+
+class PostsPage extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+        dummyData : [],
+        searchKeyWord : '',
+        posts: [],
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+    posts: dummyData
+    });
+  }
```

Comment:
once you refactor the state like how I suggested above, you won't need this componentDidMount